.travis.yml

```diff
@@ -8,11 +8,12 @@ script:
 deploy:
   - provider: pages
     skip_cleanup: true
-    github_token: $GITHUB_AUTH_TOKEN
+    github_token: $GITHUB_AUTH_SECRET
```

Comment:
Could you remind me where this is documented, please ?

Reply:
it's an environment variable in Travis